website/docs/r/terraform_version.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_terraform_version"
+sidebar_current: "docs-resource-tfe-terraform-version-x"
+description: |-
+  Manages Terraform versions
+---
+
+# tfe_terraform_version"
+
+Manages Terraform versions.
```

Comment:
I would add a little more here. How are terraform versions used? Maybe:

> Manage which versions of Terraform are available to Terraform Cloud Workspaces

Reply:
I wouldn't mention a workspace (even though it's implied that a workspace will use the version), simply because workspaces are neither an argument or attribute for a Terraform version. How about: `Manage Terraform versions available on Terraform Cloud/Enterprise` ?